plugins/MantisGraph/MantisGraph.php

```diff
@@ -65,6 +68,26 @@ function hooks() {
 	}
 
 	/**
+	 * Add the RESTful routes that are handled by this plugin.
+	 *
+	 * @param $p_event_name The event name
+	 * @param $p_event_args The event arguments
+	 * @return void
+	 */
+	function routes( $p_event_name, $p_event_args ) {
+		$t_app = $p_event_args['app'];
+		$t_app->group( '/MantisGraph', function() use ( $t_app ) {
+			$t_app->get( '/summary', function( $req, $res, $args ) {
+				if( access_has_project_level( config_get( 'view_summary_threshold' ) ) ) {
+					return $res->withStatus( 200 )->withJson( create_reporter_summary() );
+				}
+
+				return $res->withStatus( 403 );
+			} );
+		} );
+	}
```

Comment:
I have added plugin_route_group() to get the route group name for the plugin and updated documentation to provide more details and reference Slim Framework Router documentation.